src/main/java/io/codearte/jfairy/Bootstrap.java

```diff
@@ -152,4 +162,26 @@ public Fairy build() {
 		}
 	}
 
+
+	private static FairyModule getFairyModuleForLocale(Locale locale, Random random) {
```

Comment:
IMHO we should assume that someone using JFairy will pass desired module as a parameters. We should't use reflection for that.